src/Types.jl

```diff
@@ -554,7 +554,7 @@ type ModelParams{T <: Number}
     vp::VariationalParams{T}
     pp::PriorParams
     patches::Array{SkyPatch, 2}
-    tile_sources::Vector{Array{Array{Int}}}
+    tile_sources::Vector{Array{Vector{Int}, 2}}
```

Comment:
That'd be great. I'm going to try profiling the unit tests to see what else is taking so long. (And it's concerning that unit test on julia 0.5 take like 50% long than julia 0.4)